src/Goteo/Model/LegalDocumentType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Goteo\Model;
+
+use Goteo\Library\Text;
+
+use Goteo\Core\Model;
+use Goteo\Model\LegalDocumentType\NIE;
+use Goteo\Model\LegalDocumentType\CIF;
+use Goteo\Model\LegalDocumentType\NIF;
+use Goteo\Model\LegalDocumentType\VAT;
+use Goteo\Model\LegalDocumentType\PASSPORT;
+
+abstract class LegalDocumentType {
+
+    const CIF = 'cif';
+    const NIF = 'nif';
+    const NIE = 'nie';
+    const VAT = 'vat';
+    const PASSPORT = 'passport';
+    const LEGAL_DOCUMENTS = [self::CIF, self::NIF, self::NIE];
+
+    private string $document_type;
+
+    public static function create(string $document_type): LegalDocument {
```

Comment:
@davidbeig I think that the return type should be `LegalDocumentType`. Isn't it?